### PR TITLE
Removing monitor hana instance

### PR DESCRIPTION
**Pull request by pabowers**
Removing the command `az hanainstance monitor` since we have the `az sapmonitor create` command instead. The `modules_sdk` files are automatically generated from the swagger spec change and are already in the azure python sdk.

### CODE REVIEW

File: azext_hanaonazure/modules_sdk/models/_models.py [hunk 1]
```diff
@@ -489,9 +489,11 @@ class SapMonitor(Resource):
     :type hana_db_username: str
     :param hana_db_password: Database password of the HANA instance.
     :type hana_db_password: str
-    :param hana_db_password_key_vault_url: URL to the KeyVault secret that contains the HANA credentials.
```
Comment (PakDLiu) [inline azext_hanaonazure/modules_sdk/models/_models.py:492]: you might need to rebase?

**Review by PakDLiu — APPROVED** [review]

File: azext_hanaonazure/modules_sdk/models/_models.py (hunk 1 above)
Comment (pabowers) [reply azext_hanaonazure/modules_sdk/models/_models.py:492]: I already did, not sure why it is removing that

**Review by PakDLiu — APPROVED** [review]

File: azext_hanaonazure/modules_sdk/models/_models.py (hunk 1 above)
Comment (pabowers) [reply azext_hanaonazure/modules_sdk/models/_models.py:492]: The description was changed in PR https://github.com/Azure/azure-rest-api-specs/commit/79ecf24dcb870e4b293a8fef7a597a2cd1340d12#diff-5d31a3020218d4446080e6fa4795f10e, so I guess my Python SDK update caught that change

**Review by tniek — APPROVED** [review]
Looks good, thanks

**Review by Caiyeon — APPROVED** [review]

**Pull request merged**